src/routes/index.js

```diff
@@ -1,5 +1,12 @@
-const router = require("express").Router();
+const router = require('express').Router();
 
-router.use("/api", require("./api"));
+router.use('/api', require('./api'));
+
+router.get('/', (req, res) => {
```

Comment:
Please remove this endpoint here, this is not needed. We only want you to have a basic test setup.